Source/Moya+Defaults.swift

```diff
@@ -3,8 +3,12 @@ import Alamofire
 /// These functions are default mappings to `MoyaProvider`'s properties: endpoints, requests, manager, etc.
 public extension MoyaProvider {
     public final class func DefaultEndpointMapping(target: Target) -> Endpoint<Target> {
-        let url = target.baseURL.URLByAppendingPathComponent(target.path).absoluteString
-        return Endpoint(URL: url, sampleResponseClosure: {.NetworkResponse(200, target.sampleData)}, method: target.method, parameters: target.parameters)
+        if let url = target.baseURL.URLByAppendingPathComponent(target.path) {
```

Comment:
Huh! It must not be an optional. Try this?

```swift
guard let stringURL = target.baseURL.URLByAppendingPathComponent(target.path).absoluteString  else {
    fatalError(...)
}
```

`URLByAppendingPathComponent` should return a non-optional URL. Try it out and see if you can get it to work, let me know if I can help :+1: